WordPress/src/main/java/org/wordpress/android/ui/jetpack/restore/RestoreViewModel.kt

```diff
@@ -117,16 +121,26 @@ class RestoreViewModel @Inject constructor(
                     rewindId = null,
                     restoreId = null,
                     url = null,
-                    errorType = null
+                    errorType = null,
+                    optionsSelected = null,
+                    published = null
             )
         }
     }
 
-    fun onRestoreDetailsFinished(rewindId: String?, restoreId: Long?, published: Date) {
+    fun onRestoreDetailsFinished(rewindId: String?, optionsSelected: List<Pair<Int, Boolean>>?, published: Date?) {
```

Comment:
Question (❓): I know we discuss that a bit, but would you mind explaining here the difference between this and `BackupDownloadViewModel.onBackupDownloadDetailsFinished(...)`? This is the only main difference I found between the two screens so far.

Reply:
@ParaskP7 
BackupDownload details controls the submission to server for the request, where in restore it's the Warning view that controls the restore request. In order for me to pass the data along to the next view (using the wizard), it has to be parcelable. The actual requestTypes used in the API are not parcelable, so I couldn't pass those along either.

Reply:
Thanks for the explanation @zwarm , this will really help me with my `Restore` task! 🌟